### PR TITLE
Add border to notification count

### LINKED ISSUE #737: Notification badge looks a little nicer with a transparent border
screen shot 2016-11-26 at 1 52 07 pm

### PR DESCRIPTION
**Pull request by briansteffens**
Closes #737

screen shot 2017-01-12 at 11 40 56 am
screen shot 2017-01-12 at 11 41 06 am

### CODE REVIEW

File: scss/components/Header.scss [hunk 1]
```diff
@@ -91,30 +91,36 @@ $header-padding: $grid-gutter-width-base / 2;
         &:active,
         &:hover,
         &:focus {
-            background: #3c3c3c;
+            $hover-bg: #3c3c3c;
```
Comment (eatonphil) [inline scss/components/Header.scss]: Oops. I should have made a color of this earlier...

**Comment by eatonphil** [comment]
This worked out a lot better than I expected it to. How does it look for other notification count values? (Double digits, triple digits). The original code just has the circle become an oval -- which is fine. It's just the vertical alignment and that it has enough padding that was my concern.

**Comment by briansteffens** [comment]
screen shot 2017-01-12 at 11 46 39 am
screen shot 2017-01-12 at 11 46 51 am
screen shot 2017-01-12 at 11 47 01 am

**Comment by briansteffens** [comment]
I'm surprised, even with 9 digits it doesn't completely fall apart

File: scss/components/Header.scss (hunk 1 above)
Comment (eatonphil) [inline scss/components/Header.scss]: https://github.com/linode/manager/issues/966

**Pull request merged**